analyzer/tools/build-logger/src/ldlogger-tool-gcc.c

```diff
@@ -548,8 +548,11 @@ int loggerGccParserCollectActions(
   }
   else if (responseFile = getResponseFile(&action->arguments))
   {
+    LOG_INFO("Processing response file: %s", responseFile);
     loggerVectorAdd(&action->sources, responseFile);
     loggerVectorAdd(actions_, action);
+  } else {
```

Comment:
Shouldn't be the expected format be something like this?
```suggestion
  }
  else
  {
```
I was asked to change it if I remember it correctly :wink:

Reply:
Yes, you are absolutely right. Good catch :smile: